pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -13,12 +13,13 @@ const (
 	InternetReachableFromMaster status.ConditionType = "InternetReachableFromMaster"
 	InternetReachableFromWorker status.ConditionType = "InternetReachableFromWorker"
 	MachineValid                status.ConditionType = "MachineValid"
+	RedHatKeyPresent            status.ConditionType = "RedHatKeyPresent"
```

Comment:
Why is this a condition?
1. It is not extensible when the list of registries changes
2. It is not a success or failure for the list of registries to be present or not present.  It is an end user configuration decision.

It would make more sense to list the registries in a []string in the status if this was wanted.